codebase/superdarn/src.bin/tk/tool/make_snd.1.0/make_snd.c

```diff
@@ -0,0 +1,159 @@
+/* make_snd.c
+   ==========
+   Author: E.G.Thomas
```

Comment:
copyright/disclaimer/code comments

Reply:
```suggestion
Copyright (C) 2020, Dartmouth College
Author: E.G.Thomas

This program is free software: you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation, either version 3 of the License, or
(at your option) any later version.

This program is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
GNU General Public License for more details.

You should have received a copy of the GNU General Public License
along with this program.  If not, see <http://www.gnu.org/licenses/>.
```